beamer/tests/agent/unit/test_invalidate.py

```diff
@@ -77,7 +79,76 @@ def test_handle_hash_invalidated():
     assert process_event(event, context) == (True, None)
 
     # Check that a matching event invalidates the claim
```

Comment:
a different comment would be nicer. Semantically we do not see that the claim is invalidated anymore as no invalidated state exists anymore.

Reply:
btw I realise that we need to store some transaction hash same as goes for the fill in order to call l1 eventually. But we can add that in the respective issue.

Reply:
But with that tx_hash we could do the assert correctly, checking that it was invalidated.